docs/siem/detections/api/lists/api-create-list-item.asciidoc

```diff
@@ -0,0 +1,82 @@
+[[lists-api-create-list-item]]
+=== Create list item
+
+Creates a list item and associates it with the specified
+<<lists-api-create-container, list container>>.
+
+All list items in the same list container must be the same type. For example,
+each list item in an `ip` list container must exclude a specific IP address.
+
+NOTE: Before creating list items, you must create a list container.
+
+==== Request URL
+
+`POST <kibana host>:<port>/api/lists/items`
+
+==== Request body
+
+A JSON object with these fields:
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`id` |String |Unique identifier of the list item. |No, automatically created
+when it is not provided.
+|`list_id` |String |ID of the associated <<lists-api-create-container, list container>>. |Yes
+|`meta` |Object |Placeholder for metadata about the list item. |No
+|`value` |String |The value to be excluded. |Yes
```

Comment:
This is _mostly_ true here .... But lists in a way are kind of agnostic to being excluded or included and we have upcoming features where we will probably utilize them for both exclusion and inclusion based lists and it could be the same list values being used for one type and then the other.

So, it might be beneficial to somehow put on here that although they can be used within an exclusion list, they can be used for other list types or querying independent of exclusion maybe?